index.js

```diff
@@ -5,13 +5,7 @@ const solveEquationString = require('./lib/solveEquation/solveEquationString');
 const MathChangeTypes = require('./lib/MathChangeTypes');
 
 module.exports = {
-  // TODO: after deploying this and then the update to Athena.
```

Comment:
oh I did this in another PR haha, but nbd 

though make sure you do a lint to get rid of the unneeded requires